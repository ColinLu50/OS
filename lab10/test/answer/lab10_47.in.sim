43209
1424
1442
739
1424
730
