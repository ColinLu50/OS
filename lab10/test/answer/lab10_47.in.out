FCFS
696 -> 687 -> 485 -> 476 -> 310 -> 78 -> 99 -> 179 -> 373 -> 217 -> 270 -> 189 -> 10 -> 517 -> 627 -> 610 -> 255 -> 13 -> 626 -> 78 -> 146 -> 75 -> 703 -> 160 -> 200 -> 296 -> 28 -> 633 -> 78 -> 145 -> 544 -> 66 -> 628 -> 726 -> 498 -> 267 -> 383 -> 211 -> 694 -> 122 -> 303 -> 685 -> 400 -> 22 -> 130 -> 233 -> 623 -> 15 -> 342 -> 137 -> 703 -> 289 -> 662 -> 743 -> 440 -> 535 -> 151 -> 329 -> 71 -> 295 -> 623 -> 397 -> 614 -> 468 -> 27 -> 427 -> 638 -> 258 -> 310 -> 630 -> 57 -> 644 -> 586 -> 9 -> 185 -> 107 -> 727 -> 153 -> 686 -> 642 -> 686 -> 580 -> 67 -> 563 -> 647 -> 232 -> 405 -> 535 -> 440 -> 462 -> 266 -> 339 -> 688 -> 194 -> 517 -> 670 -> 348 -> 682 -> 561 -> 649 -> 378 -> 339 -> 81 -> 607 -> 706 -> 22 -> 299 -> 746 -> 67 -> 16 -> 194 -> 728 -> 40 -> 413 -> 13 -> 541 -> 125 -> 166 -> 90 -> 449 -> 605 -> 578 -> 708 -> 635 -> 75 -> 319 -> 412 -> 231 -> 444 -> 42 -> 194 -> 414 -> 328 -> 650 -> 165 -> 681 -> 343 -> 99 -> 578 -> 473 -> 28 -> 56 -> 530 -> 186 -> 20 -> 651 -> 419 -> 665 -> 544 -> 197 -> 533 -> 578 -> 495 -> 442 -> 448 -> 164 -> 103 -> 175 -> 491
43209
SSTF
696 -> 694 -> 688 -> 687 -> 686 -> 686 -> 685 -> 682 -> 681 -> 670 -> 665 -> 662 -> 651 -> 650 -> 649 -> 647 -> 644 -> 642 -> 638 -> 635 -> 633 -> 630 -> 628 -> 627 -> 626 -> 623 -> 623 -> 614 -> 610 -> 607 -> 605 -> 586 -> 580 -> 578 -> 578 -> 578 -> 563 -> 561 -> 544 -> 544 -> 541 -> 535 -> 535 -> 533 -> 530 -> 517 -> 517 -> 498 -> 495 -> 491 -> 485 -> 476 -> 473 -> 468 -> 462 -> 449 -> 448 -> 444 -> 442 -> 440 -> 440 -> 427 -> 419 -> 414 -> 413 -> 412 -> 405 -> 400 -> 397 -> 383 -> 378 -> 373 -> 348 -> 343 -> 342 -> 339 -> 339 -> 329 -> 328 -> 319 -> 310 -> 310 -> 303 -> 299 -> 296 -> 295 -> 289 -> 270 -> 267 -> 266 -> 258 -> 255 -> 233 -> 232 -> 231 -> 217 -> 211 -> 200 -> 197 -> 194 -> 194 -> 194 -> 189 -> 186 -> 185 -> 179 -> 175 -> 166 -> 165 -> 164 -> 160 -> 153 -> 151 -> 146 -> 145 -> 137 -> 130 -> 125 -> 122 -> 107 -> 103 -> 99 -> 99 -> 90 -> 81 -> 78 -> 78 -> 78 -> 75 -> 75 -> 71 -> 67 -> 67 -> 66 -> 57 -> 56 -> 42 -> 40 -> 28 -> 28 -> 27 -> 22 -> 22 -> 20 -> 16 -> 15 -> 13 -> 13 -> 10 -> 9 -> 703 -> 703 -> 706 -> 708 -> 726 -> 727 -> 728 -> 743 -> 746
1424
SCAN
696 -> 694 -> 688 -> 687 -> 686 -> 685 -> 682 -> 681 -> 670 -> 665 -> 662 -> 651 -> 650 -> 649 -> 647 -> 644 -> 642 -> 638 -> 635 -> 633 -> 630 -> 628 -> 627 -> 626 -> 623 -> 614 -> 610 -> 607 -> 605 -> 586 -> 580 -> 578 -> 563 -> 561 -> 544 -> 541 -> 535 -> 533 -> 530 -> 517 -> 498 -> 495 -> 491 -> 485 -> 476 -> 473 -> 468 -> 462 -> 449 -> 448 -> 444 -> 442 -> 440 -> 427 -> 419 -> 414 -> 413 -> 412 -> 405 -> 400 -> 397 -> 383 -> 378 -> 373 -> 348 -> 343 -> 342 -> 339 -> 329 -> 328 -> 319 -> 310 -> 303 -> 299 -> 296 -> 295 -> 289 -> 270 -> 267 -> 266 -> 258 -> 255 -> 233 -> 232 -> 231 -> 217 -> 211 -> 200 -> 197 -> 194 -> 189 -> 186 -> 185 -> 179 -> 175 -> 166 -> 165 -> 164 -> 160 -> 153 -> 151 -> 146 -> 145 -> 137 -> 130 -> 125 -> 122 -> 107 -> 103 -> 99 -> 90 -> 81 -> 78 -> 75 -> 71 -> 67 -> 66 -> 57 -> 56 -> 42 -> 40 -> 28 -> 27 -> 22 -> 20 -> 16 -> 15 -> 13 -> 10 -> 9 -> 0 -> 703 -> 706 -> 708 -> 726 -> 727 -> 728 -> 743 -> 746
1442
C-SCAN
696 -> 694 -> 688 -> 687 -> 686 -> 685 -> 682 -> 681 -> 670 -> 665 -> 662 -> 651 -> 650 -> 649 -> 647 -> 644 -> 642 -> 638 -> 635 -> 633 -> 630 -> 628 -> 627 -> 626 -> 623 -> 614 -> 610 -> 607 -> 605 -> 586 -> 580 -> 578 -> 563 -> 561 -> 544 -> 541 -> 535 -> 533 -> 530 -> 517 -> 498 -> 495 -> 491 -> 485 -> 476 -> 473 -> 468 -> 462 -> 449 -> 448 -> 444 -> 442 -> 440 -> 427 -> 419 -> 414 -> 413 -> 412 -> 405 -> 400 -> 397 -> 383 -> 378 -> 373 -> 348 -> 343 -> 342 -> 339 -> 329 -> 328 -> 319 -> 310 -> 303 -> 299 -> 296 -> 295 -> 289 -> 270 -> 267 -> 266 -> 258 -> 255 -> 233 -> 232 -> 231 -> 217 -> 211 -> 200 -> 197 -> 194 -> 189 -> 186 -> 185 -> 179 -> 175 -> 166 -> 165 -> 164 -> 160 -> 153 -> 151 -> 146 -> 145 -> 137 -> 130 -> 125 -> 122 -> 107 -> 103 -> 99 -> 90 -> 81 -> 78 -> 75 -> 71 -> 67 -> 66 -> 57 -> 56 -> 42 -> 40 -> 28 -> 27 -> 22 -> 20 -> 16 -> 15 -> 13 -> 10 -> 9 -> 0 -> 746 -> 746 -> 743 -> 728 -> 727 -> 726 -> 708 -> 706 -> 703
739
LOOK
696 -> 694 -> 688 -> 687 -> 686 -> 685 -> 682 -> 681 -> 670 -> 665 -> 662 -> 651 -> 650 -> 649 -> 647 -> 644 -> 642 -> 638 -> 635 -> 633 -> 630 -> 628 -> 627 -> 626 -> 623 -> 614 -> 610 -> 607 -> 605 -> 586 -> 580 -> 578 -> 563 -> 561 -> 544 -> 541 -> 535 -> 533 -> 530 -> 517 -> 498 -> 495 -> 491 -> 485 -> 476 -> 473 -> 468 -> 462 -> 449 -> 448 -> 444 -> 442 -> 440 -> 427 -> 419 -> 414 -> 413 -> 412 -> 405 -> 400 -> 397 -> 383 -> 378 -> 373 -> 348 -> 343 -> 342 -> 339 -> 329 -> 328 -> 319 -> 310 -> 303 -> 299 -> 296 -> 295 -> 289 -> 270 -> 267 -> 266 -> 258 -> 255 -> 233 -> 232 -> 231 -> 217 -> 211 -> 200 -> 197 -> 194 -> 189 -> 186 -> 185 -> 179 -> 175 -> 166 -> 165 -> 164 -> 160 -> 153 -> 151 -> 146 -> 145 -> 137 -> 130 -> 125 -> 122 -> 107 -> 103 -> 99 -> 90 -> 81 -> 78 -> 75 -> 71 -> 67 -> 66 -> 57 -> 56 -> 42 -> 40 -> 28 -> 27 -> 22 -> 20 -> 16 -> 15 -> 13 -> 10 -> 9 -> 703 -> 706 -> 708 -> 726 -> 727 -> 728 -> 743 -> 746
1424
C-LOOK
696 -> 694 -> 688 -> 687 -> 686 -> 685 -> 682 -> 681 -> 670 -> 665 -> 662 -> 651 -> 650 -> 649 -> 647 -> 644 -> 642 -> 638 -> 635 -> 633 -> 630 -> 628 -> 627 -> 626 -> 623 -> 614 -> 610 -> 607 -> 605 -> 586 -> 580 -> 578 -> 563 -> 561 -> 544 -> 541 -> 535 -> 533 -> 530 -> 517 -> 498 -> 495 -> 491 -> 485 -> 476 -> 473 -> 468 -> 462 -> 449 -> 448 -> 444 -> 442 -> 440 -> 427 -> 419 -> 414 -> 413 -> 412 -> 405 -> 400 -> 397 -> 383 -> 378 -> 373 -> 348 -> 343 -> 342 -> 339 -> 329 -> 328 -> 319 -> 310 -> 303 -> 299 -> 296 -> 295 -> 289 -> 270 -> 267 -> 266 -> 258 -> 255 -> 233 -> 232 -> 231 -> 217 -> 211 -> 200 -> 197 -> 194 -> 189 -> 186 -> 185 -> 179 -> 175 -> 166 -> 165 -> 164 -> 160 -> 153 -> 151 -> 146 -> 145 -> 137 -> 130 -> 125 -> 122 -> 107 -> 103 -> 99 -> 90 -> 81 -> 78 -> 75 -> 71 -> 67 -> 66 -> 57 -> 56 -> 42 -> 40 -> 28 -> 27 -> 22 -> 20 -> 16 -> 15 -> 13 -> 10 -> 9 -> 746 -> 743 -> 728 -> 727 -> 726 -> 708 -> 706 -> 703
730
